charts/internal/machine-controller-manager/seed/templates/clusterrole.yaml

```diff
@@ -19,3 +19,20 @@ rules:
   - events
   verbs:
   - "*"
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  resourceNames:
+  - machine-controller
+  - machine-controller-manager
+  verbs:
+  - get
+  - watch
+  - update
```

Comment:
@MartinWeindel Should we keep it in separate PR as this PR is for csi-driver update and the changes here  is related to MCM leader election. WDYT?

Reply:
Done, see https://github.com/gardener/gardener-extension-provider-vsphere/pull/198